content/docs/get-started/aws/review-project.md

```diff
@@ -105,16 +105,29 @@ class Program
 This Pulumi program creates an S3 bucket and exports the name of the bucket.
 
 {{% lang python %}}
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python3 -m venv venv
```

Comment:
We’ve seen many issues related to Windows not installing a `python3` binary by default. Anything we can do to make sure these docs are usable on Windows?  Perhaps just a note early on in the flow that Windows users may need to use `python` instead?

Reply:
We can be more explicit and just add instructions for Windows.

Create a virtual environment in Windows:

```
python -m venv venv
``